extensions/objects/templates/set-of-translatable-html-content-ids-editor.component.html

```diff
@@ -1,21 +1,22 @@
 <style>
-  set-of-translatable-html-content-ids-editor md-checkbox .md-label {
+  .set-of-translatable-html-content-ids-editor .mat-checkbox /deep/  .mat-checkbox-label {
```

Comment:
Can you explain this change?

Reply:
And I thought angular manages the localization of the styles to this component only. We shouldn't need the first .set-of-translatable-html-content-ids-editors class right?

Reply:
Yeah, but with this extra selector, it became more specific.